### PR TITLE
[Codegen 126] Create a function emitDoubleProp

### PR DESCRIPTION
**Pull request by rota-rossi**
## Summary:

> Create a function emitDoubleProp(name: string, optional: boolean) in parser-primitives.js. Factor out the code from [Flow](https://github.com/facebook/react-native/blob/d8ced6f8953cd896471983714e722caf50783960/packages/react-native-codegen/src/parsers/flow/components/events.js#L61-L67) and [TypeScript](https://github.com/facebook/react-native/blob/d8ced6f8953cd896471983714e722caf50783960/packages/react-native-codegen/src/parsers/typescript/components/events.js#L71-L77) into that function. Use that function in the original call site.

## Changelog:

[INTERNAL][ADDED] - emitDoubleProp in parser primitves

## Test Plan:

yarn jest packages/react-native-codegen

### CODE REVIEW

File: packages/react-native-codegen/src/parsers/typescript/components/events.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const {
   throwIfArgumentPropsAreNull,
 } = require('../../error-utils');
 const {getEventArgument} = require('../../parsers-commons');
+import {emitDoubleProp} from '../../parsers-primitives';
```
Comment (cipolleschi) [inline packages/react-native-codegen/src/parsers/typescript/components/events.js:28]: unfortunately, codegen does not support this way of importing functions.
Could you please update this with this, please:
```suggestion
const {emitDoubleProp} = require('../../parsers-primitives');
```
🙏

**Review by cipolleschi — CHANGES_REQUESTED** [review]
Hi @rota-rossi, it's amazing to see your first contribution! :D 

I left a small comments that should help you get a green status in CI. Could you please take care of updating the code with the suggestion? 🙏

File: packages/react-native-codegen/src/parsers/typescript/components/events.js (hunk 1 above)
Comment (rota-rossi) [reply packages/react-native-codegen/src/parsers/typescript/components/events.js:28]: Done! I fixed for flow and forgot to do it for typescript.

**Review by cipolleschi — APPROVED** [review]
Thank you so much! Amazing work! 😄

**Pull request closed without merging**